jep/1/README.adoc

```diff
@@ -0,0 +1,1014 @@
+  She discusses the idea with the group, determining that the idea is worth pursuing.
+  She chooses to be the "<<Sponsor>>" for this potential JEP.
+  She <<discussion, gathers initial feedback>> from the community, adjusts her design as needed,
+  records the reasons for design choices, and keeps track of differing views.
+  Daniel, an expert in the area for this JEP, volunteers to be the <<BDFL Delegate>> for this JEP.
+
+. **<<submission, Submission>>** - Andrea writes up the proposal using the JEP document template as a guide,
+  including a supporting documentation and a reference implementation if needed.
+  She then submits the JEP to the JEP editors for
+  <<approval, approval as a Draft JEP>>.
+  An editor checks the submission and determines it is ready to considered as a JEP.
+  They "approve" the submission, assigning the JEP a number, and the submission becomes a "Draft" JEP.
+
+. **<<draft, Draft Status>>** - While the JEP is a "<<draft, Draft>>", Andrea may continue to gather
+  feedback, change the proposal, and record the reasoning and differing views.
+  When she believes the design is complete and represents the consensus of the community,
+  she submits the JEP for review.
+
+. **<<review, Review>>** - The <<Reviewer>> for this JEP,
+  reviews the JEP and decides to accept it, making it an "<<accepted, Accepted>>" JEP.
+  Other possible resolutions are "<<rejected, Rejected>>",
+  "<<deferred, Deferred>>", "<<withdrawn, Withdrawn>>".
+
+. **<<accepted, Accepted Status>>** -  Andrea and other contributors
+  complete all remaining implementation related to the
+  "<<accepted, Accepted>>" JEP (code, documentation, or other changes).
+
+. **<<final, Final Status>>** - When the implementation is complete and all changes have been published
+  or otherwise incorporated into the appropriate code repositories,
+  the JEP status is changed to "<<final, Final>>".  The JEP is done.
+
+. **<<maintenance, Maintenance>>** - At some later date, the JEP may need to be updated.
+  As "Sponsor" of the JEP, Andrea makes changes as needed or hands off sponsorship to someone else.
+  Updates follow the same basic JEP workflow.
+
+IMPORTANT: The above is only a high-level overview of the JEP workflow.
+The full and complete description of the JEP workflow is provided below.
+Read the full description below before starting a JEP.
+
+[[start]]
+==== Start with an idea for Jenkins
+
+The JEP process begins with a new idea for Jenkins.
+A single JEP should contain a single key proposal or new idea.
+The more focused the JEP, the more successful it tends to be.
+The JEP editors reserve the right to reject potential JEPs
+if they appear too unfocused or too broad.
+If in doubt, sponsors should split their JEP into several well-focused ones.
+
+[NOTE]
+====
+Enhancements or patches which have a smaller impact often don't need a
+JEP can be handle via the Jenkins development workflow with a JIRA item and/or
```

Comment:
One thing that was not clarified here is whether a JEP is expected to also have a matching placeholder JIRA issue. This can be useful on occasion; for example, if a changes made as part of a JEP are discovered (after release) to cause regressions, you would want to have something to link those `regression` tickets to.